doc-content/enterprise-only/openshift/rolemapping-proc.adoc

```diff
@@ -0,0 +1,64 @@
+[id='rolemapping-proc']
+= Providing the LDAP role mapping file
+
+If you configure the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` parameter, you must provide a file that defines the role mapping. You must mount this file on all affected deployment configurations.
+
+.Procedure
+
+. Create the role mapping properties file, for example, `my-role-map`. The file must contain lines in the following format:
++
+[subs="attributes,verbatim,macros"]
+----
+ldap_role = product_role1, product_role2...
+----
++
+For example:
++
+[subs="attributes,verbatim,macros"]
+----
+admins = kie-server,rest-all,admin,kiemgmt,Administrators
+----
++
+. Create an OpenShift configuration map from the file. Run the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap ldap_role_mapping --from-file=<new_name>=<existing_name>
+----
++
+Where `new_name` is the name that the file is to have on the pods (it must be the same as the name specified in the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` file) and `existing_name` is the name of the file that you created. For example:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap ldap_role_mapping --from-file=rolemapping.properties=my-role-map
+----
++
+. Mount the configuration map on every deployment config that is configured for role mapping. The following deployment configs can be affected in this environment:
+ifeval::["{context}"=="openshift-authoring"]
+** _`myapp`_``-rhpamcentr``: {CENTRAL}
+** _`myapp`_``-kieserver``: {KIE_SERVER} 
+endif::[]
+ifeval::["{context}"=="openshift-managed"]
+** _`myapp`_``-rhpamcentrmon``: {CENTRAL} Monitoring
+** _`myapp`_``-kieserver-``_`n`_: {KIE_SERVER} number _n_. By default, the numbers are 1 and 2. 
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+** _`myapp`_``-rhdmcentr``: {CENTRAL}
+** _`myapp`_``-kieserver``: {KIE_SERVER}
+endif::[]
+ifeval::["{context}"=="openshift-immutable"]
+ifdef::PAM[]
+endif::PAM[]
+** _`myapp`_``-rhpamcentrmon``: {CENTRAL} Monitoring
+** _`myapp`_``-kieserver``: {KIE_SERVER} 
+endif::[]
+
+Where `myapp` is the application name. Sometimes, several {KIE_SERVER} deployments can be present under different application names. 
+
+For every deployment configuration, run the command:
+
+[subs="attributes,verbatim,macros"]
+----
+ oc set volume dc/<deployment_config_name> --add --type configmap --configmap-name ldap_role_mapping --mount-path=<mapping_dir> --name=ldap_role_mapping
+----
+Where `mapping_dir` is the directory name (without file name) set in the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` parameter, for example, `/opt/eap/standalone/data` .
```

Comment:
If you use this path (`/opt/eap/standalone/data`) and mount the configmap as a directory instead of a file it will override all the other files. I recommend you to change it by something different inside configuration directory. e.g. `/opt/eap/standalone/configuration/rolemapping`